### PR TITLE
Convert reStructuredText to Markdown

### PR DESCRIPTION
**Pull request by Rangi42**
Related to #1300: we weren't using any features exclusive to reStructuredText, so I converted the docs to Markdown.

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+- Ben Hetherington &lt;dev@ben-h.uk&gt;
+
+- Björn Höhrmann &lt;bjoern@hoehrmann.de&gt;
+
+- Christophe Staïesse &lt;chastai@skynet.be&gt;
+
+- David Brotz &lt;dbrotz007@gmail.com&gt;
+
+- Jakub Kądziołka &lt;kuba@kadziolka.net&gt;
```
Comment (ISSOtm) [inline CONTRIBUTORS.md:33]: I know @meithecatte has changed names. Should we follow the rename?

File: CONTRIBUTORS.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+- Vegard Nossum &lt;vegard.nossum@gmail.com&gt;
+
+- Anthony J. Bentley &lt;anthony@anjbe.name&gt;
+
+- stag019 &lt;stag019@gmail.com&gt;
+
+- Antonio Niño Díaz &lt;antonio_nd@outlook.com&gt;
+
+- Eldred "ISSOtm" Habert &lt;eldredhabert0@gmail.com&gt;
```
Comment (ISSOtm) [inline CONTRIBUTORS.md:19]:
```suggestion
- Eldred "ISSOtm" Habert &lt;me@eldred.fr&gt;
```

**Review by ISSOtm — CHANGES_REQUESTED** [review]
Thank you! I have two drive-by suggestions related to the content. Might as well while we're here?

File: CONTRIBUTORS.md (hunk 1 above)
Comment (Rangi42) [reply CONTRIBUTORS.md]: Yes, thanks!

**Review by ISSOtm — APPROVED** [review]
Thanksies~

**Pull request merged**